### DISCUSSION TITLE
Set default stroke width for Tabler Icons

### DISCUSSION THREAD

**Discussion by RowinVanAmsterdam** [tabler/tabler#1872] [General]
We use `'@tabler/icons-react'` for the iconography in one of the react projects, and was wondering if I can change the stroke width for all icons? 

Currently we have to set it manually: `<IconChevronDown stroke={1.5}  />`, but since almost every icon in the project would use a 1.5 stroke width, I would highly prefer changing the default from 2 to 1.5 somehow. Not sure if it's possible, since I can't find something about it in the docs. 

edit: Also tried to overwrite it based on the css class: tabler-icon. But it's not really working, since it gets overwritten again by the default one.

**Comment by vladstudio** [comment]
Same request.

**Comment by ignite-404** [comment]
same request

**Comment by sarendipitee** [comment]
For anyone coming across this from google, you can override the default with a global css selector:

```
svg.tabler-icon {
    stroke-width: 1;
    width: 16px;
    height: 16px;
 }
 ```

**Reply by jperez-neostella** [reply]
But then, all icons will have the same stroke, and you can't overwrite it for a specific icon with the stroke prop. Does anyone have any other solution for this?
